tests/envs/test_action_dim_check.py

```diff
@@ -70,6 +70,9 @@ def test_discrete_actions_out_of_bound(env: gym.Env):
     Args:
         env (gym.Env): the gymnasium environment
     """
+    if env.metadata.get("jax", False):
```

Comment:
wouldn't it be better to use a wrapper here?

Reply:
The issue is not actually with the wrapper, it is rather that this check from removed the environments themselves. 
We could re-add the checks in the environments then apply the wrapper but I decided not to do that for now